### DISCUSSION TITLE
English sentence for キラキラ could be improved a bit

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#14] [Errors and Corrections]
This is a very minor thing, however...

Japanese:

> 星が空にきらきら輝いている。

English:

> The stars are glittering above.

Suggested change:

> [EN] The stars are twinkling in the sky.

---

I think "above" is a bit of an unnecessary change/difference from the Japanese. "The stars are glittering in the sky" would be more clear and reflect the actual Japanese sentence much better. 

Also this is a huge nitpick, but the usage of "glitter" in the English sentence to refer to stars is technically incorrect. "Glitter" should be used for reflected light (like glittering on the surface of a body of water, etc), not light that is actually emitted directly. "Twinkle" or "sparkle" would be more appropriate and feel more natural in my opinion.

**Comment by stephenmk** [comment]
Sounds good to me. I submitted the change.
